tests/e2e/test_rosacli_user_role.go

```diff
@@ -168,39 +185,88 @@ var _ = Describe("Edit user role", labels.Feature.UserRole, func() {
 			textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
 			Expect(textData).Should(ContainSubstring("Created role"))
 			Expect(textData).Should(ContainSubstring("Successfully linked role"))
-			defer func() {
-				By("Delete user-role")
-				output, err = ocmResourceService.DeleteUserRole("--mode", "auto",
-					"--role-arn", foundUserRole.RoleArn,
-					"-y")
 
+			By("Get the user role arn")
+			userRoleList, _, err := ocmResourceService.ListUserRole()
+			Expect(err).To(BeNil())
+			foundUserRole = userRoleList.UserRole(userrolePrefix, ocmAccountUsername)
+			Expect(foundUserRole).ToNot(BeNil())
+			Expect(foundUserRole.Linded).To(Equal("No"))
+			userRoleArnsToClean = append(userRoleArnsToClean, foundUserRole.RoleArn)
+
+			By("UnLink user-role")
+			output, err = ocmResourceService.LinkUserRole("--role-arn", foundUserRole.RoleArn, "-y")
+			Expect(err).To(BeNil())
+			textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+			Expect(textData).Should(ContainSubstring("Successfully unlinked role"))
+
+			By("Get the user-role info")
+			userRoleList, _, err = ocmResourceService.ListUserRole()
+			Expect(err).To(BeNil())
+			foundUserRole = userRoleList.UserRole(userrolePrefix, ocmAccountUsername)
+			Expect(foundUserRole.Linded).To(Equal("Yes"))
+		})
+	It("can create/link/unlink/delete user-role in manual mode - [id:52693]",
+		labels.High, labels.Runtime.OCMResources,
+		func() {
+			var (
+				userrolePrefix     string
```

Comment:
i would suggest `userRolePrefix` instead of `userrolePrefix` for better readability.